app/routes/api/users.js

```diff
@@ -33,18 +85,38 @@ router.get('/:username', function(req, res, next){
 
 // create new user and authenticate with passport
 router.post('/', function(req, res, next){
-  User.register(new User({username: req.body.username, firstname: req.body.firstname, lastname: req.body.lastname, email: req.body.email}), req.body.password, function(err, user){
-
+  var hash = crypto.randomBytes(20).toString('hex');
+  User.register(new User({username: req.body.username, firstname: req.body.firstname, lastname: req.body.lastname, email: req.body.email, key: hash, confirmed: "false", permissions: "user"}), req.body.password, function(err, user){
```

Comment:
Please reformat this so it is like 
{
username: asdasd,
stuff: asdasd,
asdasd: asdasd
}